### PR TITLE
Outputs for CI

### PR DESCRIPTION
**Pull request by Drache93**
Hi all, not sure if this is a feature anyone else is interested in, but I felt it was missing in my use case.

In the event that enabled is set to false; I've added console.log to start() and stopAndPersist(). This ensures CI gets at least some form of logging for later inspection.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -147,17 +148,12 @@ class Ora {
...
+		options = options || {};
 		if (!this.enabled) {
+			console.log(`${options.symbol || ' '} ${options.text || this.text}`);
 			return this;
 		}
 
-		// TODO: Remove in the next major version
-		if (typeof options === 'string') {
-			throw new TypeError('This argument now accepts an options object, not a string');
-		}
```
Comment (sindresorhus) [inline index.js]: Don't do unrelated changes.

File: index.js [hunk 2]
```diff
@@ -102,6 +102,7 @@ class Ora {
 		}
 
 		if (!this.enabled || this.isSpinning) {
+			console.log(`${logSymbols.info} ${this.text}`);
```
Comment (sindresorhus) [inline index.js]: It should log if `this.isSpinning` is `true`.

**Comment by sindresorhus** [comment]
Yeah, makes sense. Can you add a couple of tests?

File: index.js (hunk 2 above)
Comment (Drache93) [reply index.js]: Sorry not sure what you're after. Also isn't isSpinning null at that point?

**Comment by Drache93** [comment]
Just realised CI tests were failing because XO prefers destructuring. However, Node 4 doesn't understand it (breaks on XO's internal destructuring now).

**Comment by nexdrew** [comment]
This means use-cases that do not want any text persisted now have persisted text only when run in a CI environment. IMO this is fairly annoying. It would be nice if there was an option to keep the pre-3.0.0 no-op behavior.

In the meantime, tests for ora consumers that previously did not expect any stderr output must be updated to change their expectations.
